cpp/arcticdb/version/version_map_batch_methods.hpp

```diff
@@ -231,8 +231,8 @@ inline std::vector<folly::Future<std::optional<AtomKey>>> batch_get_versions(
         } else {
             auto fut = shared_futures.find(*symbol);
             if(fut == shared_futures.end()) {
-                auto check_reload_task_fut = async::submit_io_task(CheckReloadTask{store, version_map, *symbol, it->second.load_param_});
-                auto [splitter, inserted] = shared_futures.emplace(*symbol, folly::FutureSplitter(std::move(check_reload_task_fut)));
+                auto [splitter, inserted] = shared_futures.emplace(*symbol, folly::FutureSplitter(async::submit_io_task(CheckReloadTask{store, version_map, *symbol, it->second.load_param_})));
```

Comment:
when the previous PR was submitted, there was a comment requesting splitting it in 2 statements (as is in the original code). https://github.com/man-group/ArcticDB/pull/415#discussion_r1213448013

Reply:
I didn't like that change

Reply:
We should set up clang-format and leave that to decide line-length debates IMO.

Reply:
(not in this PR, obviously)

Reply:
I didn't realise we were reverting this change in fact.

Is this intentional @willdealtry? Seems a strange change to intentionally revert in my opinion - certainly looks cleaner across two lines to me.

Reply:
I guess it's a matter of taste. To me it just seems to be introducing a variable with no purpose. We can change it back again in future if people feel strongly about it